README.md

```diff
@@ -34,39 +34,39 @@ Instantiation via `fsspec`:
 >>> import fsspec
 
 >>> # Instantiate a `hffs.HfFileSystem` object
->>> fs = fsspec.filesystem("hf://my-username/my-model-repo", repo_type="model")
+>>> fs = fsspec.filesystem("hf://model/my-username/my-model-repo")
 >>> fs.ls("")
 ['.gitattributes', 'config.json', 'pytorch_model.bin']
 
 >>> # Instantiate a `hffs.HfFileSystem` object and write a file to it
->>> with fsspec.open("hf://my-username/my-dataset-repo:/my-file-new.txt", repo_type="dataset"):
+>>> with fsspec.open("hf://dataset/my-username/my-dataset-repo:/my-file-new.txt"):
```

Comment:
Sure, it can wait!

Reply:
Actually, I still think it would be nice to promote "hf://**datasets**/my-username/my-dataset-repo:/my-file-new.txt" over "hf://**dataset**/my-username/my-dataset-repo:/my-file-new.txt" (less error-prone when copy-pasting URLs).